blockly-core/core/ui/block_space/block_space.js

```diff
@@ -189,6 +194,12 @@ Blockly.BlockSpace.createReadOnlyBlockSpace = function (container, xml) {
   }, true, true);
 
   var blockSpace = blockSpaceEditor.blockSpace;
+
+  if (opt_options.noScrolling) {
+    blockSpace.scrollbarPair.dispose();
+    blockSpace.scrollbarPair = null;
+  }
```

Comment:
Should we somehow prevent this from being created in the first place, instead of destroying it right after creating the blockSpace here?